Resources/Prototypes/Entities/Mobs/Species/reptilian.yml

```
@@ -70,20 +70,30 @@
+    displacements:
+      jumpsuit:
+        sizeMaps:
+          32:
+            sprite: _White/Mobs/Species/displacement.rsi
+            state: jumpsuit
+      shoes:
+        sizeMaps:
+          32:
+            sprite: _White/Mobs/Species/displacement.rsi
+            state: shoes
+    # WWDP-Edit-End
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)